app/jobs/endpoint_check_job.rb

```diff
@@ -0,0 +1,36 @@
+# Confirms existence of PreservedCopy on an endpoint
+# Notify ResultsRecorder, with success or failure of existence.
+class EndpointCheckJob < ApplicationJob
+  queue_as :endpoint_check
+  delegate :bucket, :bucket_name, to: PreservationCatalog::S3
+
+  # @param [PreservedCopy] verify that the archived preserved_copy exists on an endpoint
+  def perform(preserved_copy)
+    return if preserved_copy.status == PreservedCopy::UNREPLICATED_STATUS # FIXME: What to do covered in storytime
```

Comment:
whoops will remove!

Reply:
oh wait  - this is a story time question - i'll update the comment w/ the date of next story time